artifacts_test.py

```diff
@@ -115,8 +115,10 @@ def check_cluster_name(self):
                         f"Cluster name: {self.db_cluster.name}")
 
     def run_cassandra_stress(self, args: str):
```

Comment:
weird that we have another implementation also here.
We probably should unify them, but not critical for this PR.

Reply:
Since artifacts tests are not using loader nodes, aligning it won't be that straight forward, and not necessarily worth the effort.

For example calling the CassandraStress SCT class would run c-s in docker by default.